roles/tests/test_user_models.py

```diff
@@ -51,3 +51,11 @@ def __init__(self, foo_var):
 
         user_group2 = FooObject('bar')
         self.assertFalse(user_group1.__eq__(user_group2))
+
+    def test_user_group_hash(self):
+        """Test for UserGroup __hash__"""
+        user_group = UserGroup('foo1', 'bar')
+        self.assertEqual(
+            user_group.__hash__(),
+            hash((user_group.username, user_group.group_type))
```

Comment:
Don't you think this means I'm going to write unit tests for the `hash` function? 
The only thing I was planning to test was the way I was implementing the `__hash__` not the behavior of the `hash` function I call inside